pm/recipient.go

```diff
@@ -387,14 +393,22 @@ func (r *recipient) redeemWinningTicket(ticket *Ticket, sig []byte, recipientRan
 	return nil
 }
 
-func (r *recipient) rand(seed *big.Int, sender ethcommon.Address, faceValue *big.Int, winProb *big.Int, expirationBlock *big.Int, price *big.Rat) *big.Int {
+func (r *recipient) rand(seed *big.Int, sender ethcommon.Address, faceValue *big.Int, winProb *big.Int, expirationBlock *big.Int, price *big.Rat, ticketExpirationParams *TicketExpirationParams) *big.Int {
 	h := hmac.New(sha256.New, r.secret[:])
 	msg := append(seed.Bytes(), sender.Bytes()...)
 	msg = append(msg, faceValue.Bytes()...)
 	msg = append(msg, winProb.Bytes()...)
 	msg = append(msg, expirationBlock.Bytes()...)
 	msg = append(msg, price.Num().Bytes()...)
 	msg = append(msg, price.Denom().Bytes()...)
+	msg = append(
+		msg,
+		append(
+			ethcommon.LeftPadBytes(big.NewInt(ticketExpirationParams.CreationRound).Bytes(), uint256Size),
```

Comment:
Since there is already an `AuxData()` method defined for `Ticket`, maybe we can shuffle some code around and reuse that logic here. Something like:

```golang
func (t *TicketExpirationParams) AuxData() []byte {
    // Current implementation of Ticket.AuxData()
}

func (t *Ticket) AuxData() []byte {
    return &TicketExpirationParams{
        CreationRound: t.CreationRound,
        CreationRoundBlockHash: t.CreationRoundBlockHash,
    }.AuxData()
}
```

Then, we can simplify this line to:

```golang
msg = append(msg, ticketExpirationParams.AuxData())
```